### PR TITLE
Update "Spring Boot OAuth" blog post for Developer Console

### PR DESCRIPTION
**Pull request by mraible**
## Description:

Update "Spring Boot OAuth" blog post for Developer Console

### CODE REVIEW

File: _source/_posts/2017-03-21-spring-boot-oauth.md [hunk 1]
```diff
@@ -131,55 +116,13 @@ security:
       # from your Auth Server's metadata, check .well-known/openid-configuration
       # if not in .well-known/oauth-authorization-server
       userInfoUri: # userinfo_endpoint
-      preferTokenInfo: false
```
Comment (bdemers) [inline _source/_posts/2017-03-21-spring-boot-oauth.md]: why was this removed?

Comment (mraible) [reply _source/_posts/2017-03-21-spring-boot-oauth.md]: I removed it because everything still works w/o it. Regardless of whether it's there or not, the following message gets logged.

```
2017-10-11 08:29:32.315  INFO 35601 --- [nio-8080-exec-2] o.s.b.a.s.o.r.UserInfoTokenServices      : Getting user info from: https://dev-158606.oktapreview.com/oauth2/default/v1/userinfo
```

Comment (bdemers) [reply _source/_posts/2017-03-21-spring-boot-oauth.md]: cool

**Comment by bdemers** [comment]
ship it!
